exchanges/request/request.go

```diff
@@ -19,14 +19,25 @@ import (
 )
 
 // New returns a new Requester
-func New(name string, httpRequester *http.Client, l Limiter) *Requester {
-	return &Requester{
-		HTTPClient:           httpRequester,
-		Limiter:              l,
```

Comment:
Stopped exporting the `Limiter` since it is supplied as an argument, unsure if there is really a need to keep it exported (hopefully not?)